src/getHumanAge.test.js

```diff
@@ -7,4 +7,25 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should convert cat and dog ages correctly', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+  });
+
+  test('should return 0 for ages less than the first threshould', () => {
+    expect(getHumanAge(10, 10)).toEqual([0, 0]);
+  });
+
+  test('should return 1 for ages between the first and second threshould',
+    () => {
+      expect(getHumanAge(17, 17)).toEqual([1, 1]);
+    });
+
+  test('should handle large age values correctly', () => {
+    expect(getHumanAge(50, 50)).toEqual([8, 7]);
+  });
+
+  test('should round down correctly for fractional years', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
```

Comment:
This test case is a duplicate of the earlier test case for 27 years. Ensure that the expected result is updated to `[3, 2]` as per the task description.